### PR TITLE
Added jquery.tiptip@1.3 [new]

### PR DESCRIPTION
**Pull request by glensc**
plugin homepage: https://github.com/drewwilson/TipTip

   - Watch 9
   - Star 123
   - Fork 77

----

1. created temporary repo to add `package.json` (https://github.com/glensc/jquery.tiptip)
2. ran cdnjs-importer tool (`./bin/cdnjs-importer -p /home/users/glen/scm/cdnjs/cdnjs -g https://github.com/glensc/jquery.tiptip`)
3. npm test passed (`  ✓ OK » 22829 honored (18.953s)`)

will send separate PR to plugin author to include package.json and tag 1.3 version

### CODE REVIEW

File: ajax/libs/jquery.tiptip/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "1.3",
+  "licences": [
+    "MIT",
+    "GPL"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drewwilson/TipTip"
+  },
+  "filename": "README.txt",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.tiptip/package.json]: Should be a css / js file

File: ajax/libs/jquery.tiptip/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "npmName": "jquery.tiptip",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.tiptip/package.json]: Registry returned 404 for GET on https://registry.npmjs.org/jquery.tiptip

File: ajax/libs/jquery.tiptip/package.json (hunk 1 above)
Comment (glensc) [inline ajax/libs/jquery.tiptip/package.json]: it was .js file, but cdn importer seems have broken it:

https://github.com/glensc/jquery.tiptip/blob/master/package.json#L17

Comment (PeterDaveHello) [inline ajax/libs/jquery.tiptip/package.json]: hmmm ... would you please open an issue ticket for the importer?

File: ajax/libs/jquery.tiptip/package.json (hunk 2 above)
Comment (glensc) [inline ajax/libs/jquery.tiptip/package.json]: i don't think it is present in npm

**Comment by PeterDaveHello** [comment]
Once everything been fix, please squash/fixup them into only one commit, thanks.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern please also help review this PR.

**Comment by glensc** [comment]
@PeterDaveHello i did `git push -f` already to keep just one commit in PR. the previous comments will be shown as "outdated diffs"

File: ajax/libs/jquery.tiptip/package.json (hunk 2 above)
Comment (PeterDaveHello) [inline ajax/libs/jquery.tiptip/package.json]: Yes please delete it.

**Comment by PeterDaveHello** [comment]
Nope there are two commits.

**Comment by glensc** [comment]
allright, indeed, now there's only one!

**Comment by glensc** [comment]
btw, if you are worried about repo size and wanting squashed commits, then your repo is big not because number of commits, but big because large amount of files.

thus there's no size difference do i `git clone` or `git clone --depth=1`, both are over 1GiB

File: ajax/libs/jquery.tiptip/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "licences": [
+    "MIT",
+    "GPL"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drewwilson/TipTip"
+  },
+  "filename": "jquery.tipTip.minified.js",
+  "npmFileMap": [
```
Comment (PeterDaveHello) [inline ajax/libs/jquery.tiptip/package.json]: There is no npm pkg for this lib so please also remove this.

**Comment by PeterDaveHello** [comment]
Would you please also minify the css file?

**Comment by PeterDaveHello** [comment]
About the shallow clone, it will help us drop the history and then reduce the size of the repo, the history of commits and changes will be dropped. I'll show you my test result here:

```
peter@home /run/shm/cdnjs_temp $ du -d 1
818380  ./cdnjs.bare.git
716476  ./cdnjs.bare.depth.1.git
1534856 .

peter@home /run/shm/cdnjs_temp $ du -d 1 -h
800M    ./cdnjs.bare.git
700M    ./cdnjs.bare.depth.1.git
1.5G 
```
Of course it'll be smaller and save some disk space and network bandwidth.

**Comment by glensc** [comment]
about css, is there convention how to name the file?
keep the existing and add `tipTip.min.css` ?

i will squash again commits once it is ready for merge, until then i'll add commits, so it's easier to review!

**Comment by PeterDaveHello** [comment]
> about css, is there convention how to name the file?
> keep the existing and add tipTip.min.css ?

Correct!

Can you also rebase your branch with our latest master branch? Thanks!

**Comment by glensc** [comment]
rebase i'll do once all issues solved (as already said ;)

**Comment by glensc** [comment]
css added 3ebada8d3857922b7168b7b19f55194b0c61d9c2

**Comment by glensc** [comment]
rebased and squashed against f53e669. pushed over. ready to merge!

**Comment by PeterDaveHello** [comment]
Thank you @glensc

**Pull request merged**